### ISSUE TITLE
Vector Maps for maps.dymajo.com

### ISSUE THREAD

**Issue by mjdavidson** [consindo/waka#92]

**Comment by ghost** [comment]
There's a maps.dymajo.com???? Wow, I know almost nothing about this project lol

**Comment by ghost** [comment]
Error 522

**Comment by consindo** [comment]
Yeah, it's broken right now because we've run out of Azure credit. Not much we can do until the credits reset.

**Comment by ghost** [comment]
So what is the function of maps.dymajo.com

**Comment by mjdavidson** [comment]
Serving static raster .png map tiles. From a https://openmaptiles.org/ Docker Container 🙂

**Comment by mjdavidson** [comment]
https://github.com/klokantech/tileserver-gl
